### PR TITLE
Feature/hawk2 upd sp3

### PR DESCRIPTION
**Pull request by taroth21**
Done with updating - ready for review now. 
@ayoub-belarbi, @krig: Could you have a look please? There's especially one question I have: clicking the 'Plus' icon is no longer needed to add a new entry (parameter, etc.) in most Hawk2 dialogs - is that correct?  If yes, I would remove the respective sentences in the Hawk2 chapter.
But there is also a bug which might be related to that issue - see http://bugzilla.suse.com/show_bug.cgi?id=1053456.

### CODE REVIEW

**Comment by taroth21** [comment]
Merged into develop with https://github.com/SUSE/doc-sleha/commit/4f3c30cafc3b9

File: xml/ha_config_hawk2.xml [hunk 1]
```diff
@@ -269,28 +269,31 @@
...
        </listitem>
        <listitem>
         <para>
-         <guimenu>Edit Configuration</guimenu>: Allows you to edit node names
-         and attributes, resources, constraints, and tags.
+         <guimenu>Edit Configuration</guimenu>: Allows you to edit resources,
+         constraints, node names and attributes, tags, alerts, and fencing
+         topologies.
+         <remark>taroth 2017-08-10: what are the new entries, 'Alerts'
+          and 'Fencing' good for?</remark>
```
Comment (krig) [inline xml/ha_config_hawk2.xml]: Fencing is for configuring fencing topologies, see the crmsh documentation here: http://crmsh.github.io/man/#cmdhelp_configure_fencing_topology

Alerts is for configuring the new alerts feature, see the crmsh documentation here: http://crmsh.github.io/man/#cmdhelp_configure_alert

File: xml/ha_hawk2_health_i.xml [hunk 2]
```diff
@@ -26,8 +26,9 @@
  <para>
   &hawk2; provides a wizard which checks and detects issues with your cluster.
   After the analysis is complete, &hawk2; creates a cluster report with further
-  details. To verify your cluster, &hawk2; requires SSH access between your
-  cluster nodes.
+  details. To verify cluster health and generate the report, &hawk2; requires
+  passwordless SSH access between the nodes, otherwise it can only collect data
+  from the current node.
```
Comment (krig) [inline xml/ha_hawk2_health_i.xml]: Maybe adding a note that the cluster bootstrap commands automatically configure this access here would be good?

File: xml/ha_config_hawk2.xml (hunk 1 above)
Comment (taroth21) [inline xml/ha_config_hawk2.xml]: Thanks for the information! As Fate only requested a RN entry for both features, I only added this to the remark for now and added a note to myself to probably say a bit more about this for next release.

File: xml/ha_hawk2_health_i.xml (hunk 2 above)
Comment (taroth21) [inline xml/ha_hawk2_health_i.xml]: Good idea, done.

File: xml/ha_config_hawk2.xml (hunk 1 above)
Comment (krig) [inline xml/ha_config_hawk2.xml]: That sounds fine to me :)

**Pull request closed without merging**